src/main/java/org/rutebanken/tiamat/rest/graphql/mappers/SiteElementMapper.java

```diff
@@ -137,6 +141,19 @@ public boolean populate(Map input, SiteElement siteElement) {
             isUpdated = true;
         }
 
+        Optional<List<LocalService>> localServices = localServicesMapper.map(input);
+        if(localServices.isPresent()) {
+            if (siteElement instanceof Site_VersionStructure) {
```

Comment:
You can use if(siteElement instance Site_VersionStructure siteVersionStructure) {
siteVersionStructure.setLocalServices(localServices.get())